pegasus/router.rb

```diff
@@ -316,7 +316,7 @@ def self.load_configs_in(dir)
   not_found do
     status 404
     path = resolve_template('views', settings.template_extnames, '/404')
-    document path
+    (document path).tap{dont_cache}
```

Comment:
Changed the syntax according to your suggestion, I think it's more natural too. Thanks!